src/pages/Offer/sections/LegalText.tsx

```diff
@@ -19,12 +31,66 @@ const LegalText = styled('div')({
   maxWidth: LEGALWIDTH,
 })
 
-export const Legal: React.SFC = () => (
+const Link = styled('a')({
+  color: colors.PURPLE,
+  textDecoration: 'none',
+})
+
+export const Legal: React.SFC<TermsProps> = ({ insuranceType }) => (
   <Container>
     <LegalText>
-      <TranslationsConsumer textKey="OFFER_FOOTER_LEGAL_TEXT">
+      <TranslationsPlaceholderConsumer
+        textKey="OFFER_FOOTER_LEGAL_TEXT"
+        replacements={{
+          gdpr: (
+            <TranslationsConsumer textKey="CHAT_INPUT_PERSONAL_DATA_LINK">
+              {(url) => (
+                <Link href={url} target="_blank">
+                  <TranslationsConsumer textKey="OFFER_FOOTER_GDPR">
+                    {(t) => t}
+                  </TranslationsConsumer>
+                </Link>
+              )}
+            </TranslationsConsumer>
+          ),
+          preBuy: (
+            <TranslationsConsumer
+              textKey={
+                isApartmentOwner(insuranceType)
+                  ? 'TERMS_PDF_PREBUY_OWNER_URL'
+                  : 'TERMS_PDF_PREBUY_RENT_URL'
+              }
+            >
+              {(url) => (
+                <Link href={url} target="_blank">
+                  <TranslationsConsumer textKey="OFFER_FOOTER_GDPR">
+                    {(t) => t}
+                  </TranslationsConsumer>
+                </Link>
+              )}
+            </TranslationsConsumer>
+          ),
+          terms: (
+            <TranslationsConsumer
+              textKey={
+                isApartmentOwner(insuranceType)
+                  ? 'TERMS_PDF_INSURANCE_OWNER_URL'
+                  : 'TERMS_PDF_INSURANCE_RENT_URL'
+              }
+            >
+              {(url) => (
+                <Link href={url} target="_blank">
```

Comment:
Make sure to set `rel="noreferrer noopener"` for all links that have `target="_blank"`